From 469c4c5ca44252ae52944a504fbbd204a25606d1 Mon Sep 17 00:00:00 2001
From: Paul Barker <paul@betafive.co.uk>
Date: Mon, 21 Oct 2019 10:40:37 +0100
Subject: [PATCH 3/4] bcm2835-bootfiles: Clarify license conditions

We define a new 'Broadcom-RPi' license and include the full text in our
custom-licenses directory.

Signed-off-by: Paul Barker <paul@betafive.co.uk>
---
 files/custom-licenses/Broadcom-RPi         | 29 ++++++++++++++++++++++
 recipes-bsp/bootfiles/bcm2835-bootfiles.bb |  2 +-
 2 files changed, 30 insertions(+), 1 deletion(-)
 create mode 100644 files/custom-licenses/Broadcom-RPi

diff --git a/files/custom-licenses/Broadcom-RPi b/files/custom-licenses/Broadcom-RPi
new file mode 100644
index 0000000..6f3baa0
--- /dev/null
+++ b/files/custom-licenses/Broadcom-RPi
@@ -0,0 +1,29 @@
+Copyright (c) 2006, Broadcom Corporation.
+Copyright (c) 2015, Raspberry Pi (Trading) Ltd
+All rights reserved.
+
+Redistribution.  Redistribution and use in binary form, without
+modification, are permitted provided that the following conditions are
+met:
+
+* This software may only be used for the purposes of developing for,
+  running or using a Raspberry Pi device.
+* Redistributions must reproduce the above copyright notice and the
+  following disclaimer in the documentation and/or other materials
+  provided with the distribution.
+* Neither the name of Broadcom Corporation nor the names of its suppliers
+  may be used to endorse or promote products derived from this software
+  without specific prior written permission.
+
+DISCLAIMER.  THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND
+CONTRIBUTORS "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING,
+BUT NOT LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND
+FITNESS FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE
+COPYRIGHT OWNER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT,
+INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING,
+BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS
+OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND
+ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR
+TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE
+USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH
+DAMAGE.
diff --git a/recipes-bsp/bootfiles/bcm2835-bootfiles.bb b/recipes-bsp/bootfiles/bcm2835-bootfiles.bb
index 0d636f0..aada918 100644
--- a/recipes-bsp/bootfiles/bcm2835-bootfiles.bb
+++ b/recipes-bsp/bootfiles/bcm2835-bootfiles.bb
@@ -1,5 +1,5 @@
 DESCRIPTION = "Closed source binary files to help boot the ARM on the BCM2835."
-LICENSE = "Proprietary"
+LICENSE = "Broadcom-RPi"
 
 LIC_FILES_CHKSUM = "file://LICENCE.broadcom;md5=4a4d169737c0786fb9482bb6d30401d1"
 
-- 
2.23.0

